pkg/local_workflows/local_models/transform.go

```diff
@@ -132,7 +140,7 @@ func mapSuppressions(res sarif.Result) *TypesSuppression {
 			},
 		},
 		Justification: &suppression.Justification,
-		Kind:          "ignored",
+		Status:        TypesSuppressionStatus(status),
```

Comment:
Question: was kind removed from the data structure?

Reply:
Yes, but more like corrected rather than removed.

The "kind" implemented in LFM refers to the SARIF's "state"/"status" and so this change was made. The SARIF's "kind" was/is hardcoded in the `local_findings.sarif.tmpl` as it was never used in LFM.